Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/NativeIndex.kt

```diff
@@ -85,15 +85,17 @@ interface TypeDeclaration {
     val location: Location
 }
 
+sealed class StructMember
+
 /**
  * C struct field.
  */
-class Field(val name: String, val type: Type, val offset: Long, val typeAlign: Long)
+class Field(val name: String, val type: Type, val offset: Long, val typeSize: Long, val typeAlign: Long) : StructMember()
 
 val Field.isAligned: Boolean
     get() = offset % (typeAlign * 8) == 0L
 
-class BitField(val name: String, val type: Type, val offset: Long, val size: Int)
+class BitField(val name: String, val type: Type, val offset: Long, val size: Int) : StructMember()
```

Comment:
Shalln't `name`, `type` and `offset` belong to `StructMember`?

Reply:
Done.